services/docker-compose.yml

```diff
@@ -393,6 +393,12 @@ services:
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL}
+      - RESOURCE_USAGE_TRACKER_S3=${RESOURCE_USAGE_TRACKER_S3}
+      - S3_ACCESS_KEY=${S3_ACCESS_KEY}
+      - S3_BUCKET_NAME=${S3_EXPORT_CSV_BUCKET_NAME}
+      - S3_ENDPOINT=${S3_ENDPOINT}
+      - S3_SECRET_KEY=${S3_SECRET_KEY}
+      - S3_SECURE=${S3_SECURE}
```

Comment:
This confuses me a bit: I'm not that sure I'd like to have them so mixed up.
Either `RESOURCE_USAGE_TRACKER_S3` contains all the S3 configuration or you pass in new `S3_...` values.
For sure passing `S3_ACCESS_KEY` and the rest of them is a recipe for disaster.

Reply:
Yes, there will be a unique S3 access for export file usecase, already its there: 
```
      - S3_ACCESS_KEY=${S3_EXPORT_ACCESS_KEY}
      - S3_BUCKET_NAME=${S3_EXPORT_BUCKET_NAME}
      - S3_ENDPOINT=${S3_EXPORT_ENDPOINT}
      - S3_SECRET_KEY=${S3_EXPORT_SECRET_KEY}
      - S3_SECURE=${S3_EXPORT_SECURE}
```